src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets

```diff
@@ -51,7 +58,7 @@
   <Target
     Name="_AddDesktopReferences"
     AfterTargets="ResolveLockFileReferences"
-    Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework'">
+    Condition="'$(_MvcViewCompilationAddDesktopReferences)'!='false' AND '$(TargetFrameworkIdentifier)'=='.NETFramework'">
```

Comment:
This target gets called too early and results in `_MvcViewCompilationBinaryPath` being not set by common-testapps. Results in warnings when running tests. We'll use MusicStore to get coverage for this.